### PR TITLE
log ouptut for HQtoWarehouse ETL

### PR DESCRIPTION
**Pull request by snopoke**
##### SUMMARY
Adds some logging for warehouse ETL processes. This will help when looking at the airflow logs.

### CODE REVIEW

**Review by mkangia — APPROVED** [review]
Don't have good context so would defer to @calellowitz for review from implementation perspective, coding wise looks good.

File: corehq/warehouse/loaders/synclog.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ class SyncLogStagingLoader(HQToWarehouseETLMixin, BaseStagingLoader):
     """
     slug = SYNCLOG_STAGING_SLUG
     model_cls = SyncLogStagingTable
+    log_frequency = 10000
```
Comment (calellowitz) [inline corehq/warehouse/loaders/synclog.py]: Are there many fewer synclogs than forms?

**Review by calellowitz — COMMENTED** [review]
One question about the default frequency but this all looks good

File: corehq/warehouse/loaders/synclog.py (hunk 1 above)
Comment (snopoke) [reply corehq/warehouse/loaders/synclog.py]: yes, an order of magnitude less

**Pull request merged**